config/live/.htaccess

```
@@ -104,17 +104,19 @@ RewriteRule ^cawemo/(.*) /cawemo/%1 [R=307,L]
 RewriteRule ^enterprise/previous-downloads.html$ /enterprise/download/ [R=301,L]
 RewriteRule ^(latest|7.3|7.2|7.1|7.0)/enterprise /enterprise/ [R=301,L]
 
-# Redirect /get-started/cmmnXX to /get-started/cmmn11 if folder cmmnXX does not exist
 # Redirect /get-started/dmnXX to /get-started/dmn11 if folder dmnXX does not exist
-RewriteCond %{REQUEST_URI} ^/get-started/(dmn|cmmn)(\d+)
-RewriteCond %{DOCUMENT_ROOT}/get-started/%1%2 !-d
-RewriteRule ^get-started/[^/]+(.*) /get-started/%111$1 [R=307,L]
+RewriteCond %{REQUEST_URI} ^/get-started/dmn(\d+)
+RewriteCond %{DOCUMENT_ROOT}/get-started/dmn%1 !-d
+RewriteRule ^get-started/[^/]+(.*) /get-started/dmn11$1 [R=307,L]
```

Works! With the move to dmn 1.2, we changed the dmn path to just `dmn`, so the current redirect path is `dmnXX > dmn11 > dmn`. I think we can redirect it directly to the correct path:

```suggestion
RewriteRule ^get-started/[^/]+(.*) /get-started/dmn$1 [R=307,L]
```

---

I updated the redirect with https://github.com/camunda/camunda-docs-static/pull/175/commits/e6f160a1acb735a3981b8d549a355f028070db33. I decided to leave the alias in index.md, because that also handles deep links (e.g. https://docs.camunda.org/get-started/dmn11/install/)